src/quo2/components/dividers/divider_label.cljs

```diff
@@ -36,18 +36,18 @@
         padding-top                 (if increase-padding-top? 16 8)
         text-and-icon-color         (if dark? colors/neutral-40 colors/neutral-50)
         counter-text-color          (if dark? colors/white colors/neutral-100)]
-    [rn/touchable-without-feedback
-     {:on-press on-press}
+    [rn/touchable-without-feedback {:on-press on-press}
      [rn/view
       {:accessible          true
        :accessibility-label :divider-label
-       :style               (merge {:border-top-width   1
-                                    :border-top-color   border-and-counter-bg-color
-                                    :padding-top        padding-top
-                                    :padding-bottom     padding-bottom
-                                    :padding-horizontal 16
-                                    :align-items        :center
-                                    :flex-direction     :row}
+       :style               (merge {:border-top-width 1
+                                    :border-top-color border-and-counter-bg-color
+                                    :padding-top      padding-top
+                                    :padding-bottom   padding-bottom
+                                    :padding-left     16
+                                    :padding-right    16
```

Comment:
`:padding-horizontal` was set, and since these styles are not correct (in figma the padding right is 20 for this component) I'm overriding the padding right.

The problem is the padding-left & right are ignored in React Native if `padding-horizontal` is set. So the component looks the same but allows to override those paddings with this change

![image](https://github.com/status-im/status-mobile/assets/90291778/f57a0673-48a3-45c1-a87c-6c2ab40fdd6d)